### PR TITLE
[SeaTunnel #1037] Add InfluxDB Flink Sink

### LINKED ISSUE #1037: [Feature][Connectors] Add InfluxDB Flink Sink
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

# Sink plugin: InfluxDB

## Description

Write data to InfluxDB.

## Options

| name        | type           | required | default value |
| ----------- | -------------- | -------- | ------------- |
| server_url  | `String`       | yes      | -             |
| database    | `String`       | yes      | -             |
| measurement | `String`       | yes      | -             |
| tags        | `List<String>` | yes      | -             |
| fields      | `List<String>` | yes      | -             |

### server_url [`String`]

The URL of InfluxDB Server.

### datasource [`String`]

The DataSource name in InfluxDB.

### measurement [`String`]

The Measurement name in InfluxDB.

### tags [`List<String>`]

The list of Tag in InfluxDB.

### fields [`List<String>`]

The list of Field in InfluxDB.

## Example

```hocon
InfluxDbSink {
  server_url = "http://127.0.0.1:8086/"
  database = "influxdb"
  measurement = "m"
  tags = ["country", "city"]
  fields = ["count"]
}
```

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by asdf2014**
## Purpose of this pull request

Fixes #1037

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -427,6 +428,12 @@
                 <artifactId>config</artifactId>
                 <version>${config.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>org.influxdb</groupId>
+                <artifactId>influxdb-java</artifactId>
+                <version>${influxdb-java.version}</version>
+            </dependency>
```
Comment (wuchunfu) [inline pom.xml:436]: This dependency seems to be the MIT open source license, does it conflict with the Apache open source license? If there is no conflict, whether to add relevant instructions in the declaration file, please check. @CalvinKirs

Comment (CalvinKirs) [reply pom.xml:436]: This is allowed, you can refer to this https://apache.org/legal/resolved.html

Comment (wuchunfu) [reply pom.xml:436]: > This is allowed, you can refer to this https://apache.org/legal/resolved.html

OK, Thanks

**Comment by garyelephant** [comment]
LGTM

**Review by garyelephant — APPROVED** [review]

**Pull request merged**